user-group/humhub-mass-import-inline.html

```diff
@@ -0,0 +1,164 @@
+                <div class="content" style="box-sizing:border-box;display:block;Margin:0 auto;max-width:580px;padding:10px;">
+                    <!-- START CENTERED WHITE CONTAINER -->
+                    <span class="preheader" style="color:transparent;display:none;height:0;max-height:0;max-width:0;opacity:0;overflow:hidden;mso-hide:all;visibility:hidden;width:0;">Your Blackboard Ally User Group account is ready.</span>
+                    <table class="main" style="border-collapse:separate;mso-table-lspace:0pt;mso-table-rspace:0pt;background-color:#FFFFFF;border-radius:3px;width:100%;">
+                        <!-- START MAIN CONTENT AREA -->
+                        <tr>
+                            <td class="wrapper" style="font-family:sans-serif;font-size:14px;vertical-align:top;box-sizing:border-box;text-align:center;">
+                                <table border="0" cellpadding="0" cellspacing="0" style="border-collapse:separate;mso-table-lspace:0pt;mso-table-rspace:0pt;width:100%;">
+                                    <tr>
+                                        <td class="header" style="font-family:sans-serif;font-size:14px;vertical-align:top;background-color:#191E25;box-sizing:border-box;padding:20px;">
+                                            <h1 style="color:#191F25;font-family:sans-serif;font-weight:normal;line-height:1.4;margin:0;text-align:center;">
+                                                <img alt="Ally logo" height="70px" src="http://ally.ac/assets/img/ally-logo.png" title="Ally" style="border:none;-ms-interpolation-mode:bicubic;max-width:100%;"/></h1>
+                                                <h2 style="color:#191F25;font-family:sans-serif;font-weight:normal;line-height:1.4;margin:0;color:#FFFFFF;font-size:34px;font-weight:300;text-align:center;text-transform:capitalize;">User Group Space</h2>
+                                            </td>
+                                        </tr>
+                                    </table>
+                                    <table border="0" cellpadding="0" cellspacing="0" style="border-collapse:separate;mso-table-lspace:0pt;mso-table-rspace:0pt;width:100%;">
+                                        <tr>
+                                            <td class="account" style="font-family:sans-serif;font-size:14px;vertical-align:top;padding-top:32px;text-align:center;line-height:2em;">
+                                                <h3 style="font-family:sans-serif;font-weight:normal;line-height:1.4;margin:0;color:#191F25;font-size:34px;">Your account is ready</h3>
+                                                <p style="font-family:sans-serif;font-size:14px;font-weight:normal;margin:0;">Your username: <strong>NicolaasMatthijs@fronteer.io</strong></p>
+                                                <p style="font-family:sans-serif;font-size:14px;font-weight:normal;margin:0;">Your temporary password: <strong>IBpvjp/Zkkd97OKt</strong></p>
```

Comment:
True, we can do another review once they have integrated it and I can ask him to do a Litmus test.